### PR TITLE
Change picture hash in URL while slideshow

### PR DESCRIPTION
**Pull request by MorrisJobke**
original changeset by @vsaffonov in https://github.com/owncloud/apps/pull/1572

@jancborchardt  @DeepDiver1975

### CODE REVIEW

**Comment by jancborchardt** [comment]
@icewind1991 can you check this out?

**Comment by jancborchardt** [comment]
@MorrisJobke can you rebase this?

@icewind1991 please also have a look at this.

File: js/slideshow.js [hunk 1]
```diff
@@ -22,15 +22,24 @@ jQuery.fn.slideShow = function (container, start, options) {
 	jQuery.fn.slideShow.current = start;
 	if (settings.images) {
 		images = settings.images;
+		// TODO: we need to fill the dataPath variable somehow here
```
Comment (MorrisJobke) [inline js/slideshow.js:25]: I don't have the time ti fix this here. Can somebody have a look?

**Comment by jancborchardt** [comment]
@vsaffonov @PVince81 check this out – the Pictures app needs some love.

**Comment by icewind1991** [comment]
Fixed with 4cf73a9352968a477174169de318b43214452fba

**Pull request closed without merging**